tokio-console/tests/ui/mod.rs

```diff
@@ -0,0 +1,23 @@
+use std::env;
```

Comment:
Generally I think it's easier to read if we use a single file module, rather than a directory.

Could you please change this file to `tokio-console/tests/ui.rs`.

Reply:
Sounds good. I will move it out.